### PR TITLE
updated so cloud tests pass

### PR DESCRIPTION
**Pull request by anneCarlson**

### CODE REVIEW

File: spec/indico_batch_spec.rb [hunk 1]
```diff
@@ -21,7 +21,9 @@
   it 'should access a private cloud' do
     expected_keys = Set.new(%w(Conservative Green Liberal Libertarian))
     data = ['Guns don\'t kill people.', ' People kill people.']
+    Indico.cloud_protocol = 'http://'
```
Comment (madisonmay) [inline spec/indico_batch_spec.rb]: Just leave a quick comment here to explain why this is needed...

File: spec/indico_spec.rb [hunk 2]
```diff
@@ -18,7 +18,9 @@
   it 'should tag text with correct political tags' do
     expected_keys = Set.new(%w(Conservative Green Liberal Libertarian))
     data = 'Guns don\'t kill people. People kill people.'
+    Indico.cloud_protocol = 'http://'
```
Comment (madisonmay) [inline spec/indico_spec.rb]: Ditto.

File: spec/indico_batch_spec.rb (hunk 1 above)
Comment (anneCarlson) [inline spec/indico_batch_spec.rb]: Cool, I'll get to this soon!

Comment (madisonmay) [inline spec/indico_batch_spec.rb]: I've got a bit more downtime while things are uploading, so I'll snag this edit!

**Comment by madisonmay** [comment]
LGTM -- rebase and merge at will...

**Pull request closed without merging**